src/lib/server.ts

```diff
@@ -210,15 +215,27 @@ export class Client {
       if (err instanceof BaseError) {
         this.logger.error("Message error", err);
         const { errorCode, name, message, stack, ...args } = err;
-        return this.sendResultError(msg.messageId, errorCode, args);
+        if (this.schemaVersion < 32) {
+          this.sendResultZWaveError(
+            msg.messageId,
+            -1 as any,
```

Comment:
I suppose we don't expect this to be hit? Maybe add a comment to explain this?

Reply:
no it could be hit, but what we are doing is finding a way to pass the message out because we didn't have that as an attribute before. So we are reusing the ZWaveError, and using `-1` so it doesn't overlap with any valid ZwaveErrorCodes